specs/_features/eip7732/beacon-chain.md

```diff
@@ -1029,6 +1029,13 @@ def process_attestation(state: BeaconState, attestation: Attestation) -> None:
                 # Update the builder payment weight
                 if flag_index == TIMELY_HEAD_FLAG_INDEX and is_attestation_same_slot(state, data):
                     payment.weight += state.validators[index].effective_balance
+                elif flag_index == TIMELY_TARGET_FLAG_INDEX and is_attestation_same_slot(state, data):
```

Comment:
This will add the payment twice, once for the `TIMELY_HEAD_FLAG_INDEX` and another one for the `TIMELY_TARGET_FLAG_INDEX`.  Perhaps the simplest is to just remove the if/elif statements and just do this:
```
if weight == 0 and is_attestation_same_slot(state, data):
                    is_correct_block = data.beacon_block_root == get_block_root_at_slot(state, Slot(data.slot))
                    is_correct_payload = data.index == 0  # Same-slot attestations must have index=0

                    if is_correct_block and is_correct_payload:
                        weight = state.validators[index].effective_balance
``` 
And then outside of the flag loop `payment.weight += weight`